File: QE_FST_Selenium_python/Activity13_Tables1.py
from selenium import webdriver
from selenium.webdriver.common.by import By
with webdriver.Firefox() as driver:
    driver.get("https://training-support.net/webelements/tables")
    print(driver.title)
    rows=driver.find_elements(By.XPATH,"//table/tbody/tr")
    print(len(rows))
    columns=driver.find_elements(By.XPATH,"//table/tbody/tr[1]/td")
    print(len(columns))
    ThirdRow=driver.find_elements(By.XPATH,"//table/tbody/tr[3]/td")
    for val in ThirdRow:
        print(val.text)
    data=driver.find_element(By.XPATH,"//table/tbody/tr[2]/td[2]")
    print(data.text)

File: QE_FST_Selenium_java/Activity13_Tables1.java
package demos;

import java.util.List;

import org.openqa.selenium.By;
import org.openqa.selenium.WebDriver;
import org.openqa.selenium.WebElement;
import org.openqa.selenium.firefox.FirefoxDriver;

public class Activity13_Tables1 {
    public static void main(String[] args) {
        WebDriver driver=new FirefoxDriver();
        driver.get("https://training-support.net/webelements/tables");
        System.out.println(driver.getTitle());
        List<WebElement> rows=driver.findElements(By.xpath("//table/tbody/tr"));
        System.out.println(rows.size());
        List<WebElement> columns=driver.findElements(By.xpath("//table/tbody/tr[1]/td"));
        System.out.println(columns.size());
        List<WebElement> row3=driver.findElements(By.xpath("//table/tbody/tr[3]/td"));
        for(WebElement data:row3)
        {
            System.out.println(data.getText());
        }
        WebElement cellValue=driver.findElement(By.xpath("//table/tbody/tr[2]/td[2]"));
        System.out.println(cellValue.getText());
        driver.quit();


    }
}
